gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-6.7-20200905230036+0000-bin.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-6.7-20200906230029+0000-bin.zip
```

Comment:
This change also revealed a bug somewhere in how precompiled script plugin state is cached.
This change fails to compile using the same Gradle wrapper due to state conflicts in `~/.gradle/caches/6.7-20200905230036+0000` and `buildSrc/.gradle/caches/6.7-20200905230036+0000`.
As a workaround, I bumped the wrapper here to the latest.
We may want to first replicate the bug in a test and fix it - I hope I'll get some time to dig into it this week.

Reply:
Lets open an issue for the bug. I think we can still do this change with the wrapper bump to not block this PR on the bugfix.

Reply:
https://github.com/gradle/gradle/issues/14446